### DISCUSSION TITLE
[TimeSeries Area chart] REGRESSION : New area chart don't have the "stack / expand" option

### DISCUSSION THREAD

**Discussion by ValentinC-BR** [apache/superset#21191] [Q&A / Help]
### Description

The current (deprecated) area chart has a very useful option : "Extra controls"

![image](https://user-images.githubusercontent.com/79460908/176386406-1794a803-6629-4811-ab70-f28610141a97.png)

This option allows the reader to switch between stacked and 100% stacked values : 
![image](https://user-images.githubusercontent.com/79460908/176386689-84729cc1-1d0e-4c3c-96ed-a56e2f1866aa.png)

This option is not present on the new "TimeSeries area chart, that is supposed to replace the current area chart.

**Comment by stephenLYZ** [comment]
@ValentinC-BR Hi, actually in this [PR](https://github.com/apache/superset/pull/16493) we have introduced extra control, can you try to use the latest master branch?

**Comment by ValentinC-BR** [comment]
Is it the latest stable v1.5 release ?

**Comment by ValentinC-BR** [comment]
@stephenLYZ, I meant "Is this PR merged in an official release ? Or was it merged to master, but never been part of any release candidate yet ?

**Comment by stephenLYZ** [comment]
@ValentinC-BR Hi,  It would be included in [2.0](https://github.com/apache/superset/pull/20490) release.
